### ISSUE TITLE
Suggestion - Offline Notice/Icon

### ISSUE THREAD

**Issue by bossloper** [evolution-cms/evolution#734]
If a site is offline (System Config -> Site -> Site Status) it might be useful to have a visual notice/icon somewhere on the manager interface to warn the site is currently offline. As a logged in manager you can always see the site so it is not always obvious it might be offline.

**Comment by pmfx** [comment]
Good idea!

**Comment by bossloper** [comment]
ref #735

**Comment by nick0** [comment]
Great idea @bossloper . 

This happens in WordPress with some of the better maintenance mode plugins and is a handy visual in the admin of those sites.

It would be nice if a word (eg offline (or similar)) could accompany the icon so it is a little clearer what the warning is for (otherwise it could mean hacking or a nuclear meltdown with that icon :)).

**Comment by bossloper** [comment]
The icon would have popup text 'Site Offline!' when you hover over it.
Isn't the world heading for nuclear meltdown anyway?! :-)

**Comment by Pathologic** [comment]
https://github.com/evolution-cms/evolution/pull/735 merged

**Comment by Pathologic** [comment]
https://github.com/evolution-cms/evolution/pull/735 reverted

**Comment by nick0** [comment]
I have a lot of real estate on my screen and like the idea of the symbol being visible on every page, not just the dashboard. The original application is best.

**Comment by bossloper** [comment]
How about something like this?

![untitled](https://user-images.githubusercontent.com/1744687/41763636-a97acaa6-75f6-11e8-8943-236835087a3c.png)

![untitled2](https://user-images.githubusercontent.com/1744687/41763640-ac4e917c-75f6-11e8-93e3-0e8c067b0624.png)

**Comment by fourroses666** [comment]
Or over the (view site) monitor icon?

**Comment by bossloper** [comment]
Maybe. I put it on the config because that is where you would go to switch the site back online, though there are good arguments for either!

**Comment by 64j** [comment]
Suggest more options


![image](https://user-images.githubusercontent.com/4653189/41764518-0a348bda-760a-11e8-9da5-24b60fa86f23.png)

![image](https://user-images.githubusercontent.com/4653189/41764881-1d917b74-760b-11e8-9f0a-ec14f93b1ed5.png)

**Comment by pmfx** [comment]
@64j yes, red icon and alert on the dashboard looks good and can both be used. Maybe just text on the button can be different than "Enable"? "Publish site"? "Go online"?

**Comment by bossloper** [comment]
All good to me, The colored preview icon is good in combination with the dashboard message. Agree with @pmfx the text could be 'Set to Online' or similar.

**Comment by 64j** [comment]
@pmfx 
Yes, what text to write is no longer a problem.
I suggested options as it might look.
